index.html

```diff
@@ -57,27 +61,27 @@ <h2>Here's a time line of Dr. Borlaug's life:
           <li><b>2009</b> - dies at the age of 95.</li>
```

Comment:
Please replace b  element with strong element or font weight css property. b element has no semantic meaning and it will be no  importance for web user using screen reader